services/streammanager/lambda/lambdamanager_test.go

```diff
@@ -0,0 +1,154 @@
+package lambda
```

Comment:
[minor & opinionated] Testing from the perspective of a consumer. Requires exposure of LambdaClient.

```suggestion
package lambda_test
```